CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### Enhancements
 
 * **Duplicate CLI param check** Given that many of the options associated with the `Click` based cli ingest commands are added dynamically from a number of configs, a check was incorporated to make sure there were no duplicate entries to prevent new configs from overwriting already added options.
+* **Remove pdfminer from inside tables** Previously when using `hi_res` some elements where extracted using pdfminer too, so we removed pdfminer from the tables pipeline to avoid duplicated elements. Now 
```

Comment:
Did you mean to say more? "Now".. all by its lonesome at the end :)

Reply:
Thanks, I deleted that last part.